.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,29 @@
+name: CI/CD
+
+on:
+    push:
+        branches:
+            - main
+
+jobs:
+  release:
+      runs-on: ubuntu-latest
+      if: github.ref == 'refs/heads/main'
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v4
+
+        # Setup .npmrc file to publish to npm
+        - name: Setup npmrc
+          uses: actions/setup-node@v4
+          with:
+            node-version: '20.x'
+            registry-url: 'https://registry.npmjs.org'
+
+        - name: install
+          run: npm install
+
+        - name: Publish
```

Comment:
Before publishing, we need to do some kind of versioning here. Some other projects in here use [semantic-release](https://www.npmjs.com/package/semantic-release), but it could really be any tool.

Reply:
i see a couple flavours of this being used in our projects, one which fetches versions before (https://github.com/vertigis/vertigis-web-sdk/blob/9fa6e70b41d39d2d63b68d66eb85b87c4adafeef/.github/workflows/ci-cd.yml#L41)  and others using a release artifact (https://github.com/vertigis/workflow-activities-fme/blob/fcf29be9b1869f612b1d1af5338376256595b0c7/.github/workflows/ci-cd.yml#L68). Is there a preference as to which we want to use?

Reply:
I'm more familiar with the structure in workflow-activities-fme, so I'd try that first. But really, it's whatever you're comfortable with. I think either way, we'll need to update the PR title to match the commit messaging needed for semantic-release.

Reply:
Sounds good, fme flavor has been added

Reply:
On second thought.... looking at this build, I don't think the fme flavour will work here-- we aren't creating a release artifact in a previous build stage. The web-sdk flavour with the git checkout should work fine though.